core/src/main/java/org/elasticsearch/rest/BytesRestResponse.java

```diff
@@ -127,16 +138,13 @@ private static XContentBuilder convert(RestChannel channel, RestStatus status, T
             builder.startArray();
             for (ElasticsearchException rootCause : rootCauses){
                 builder.startObject();
-                rootCause.toXContent(builder, new ToXContent.DelegatingMapParams(Collections.singletonMap(ElasticsearchException.REST_EXCEPTION_SKIP_CAUSE, "true"), channel.request()));
+                rootCause.toXContent(builder, new ToXContent.DelegatingMapParams(Collections.singletonMap(ElasticsearchException.REST_EXCEPTION_SKIP_CAUSE, "true"), params));
```

Comment:
sorry.. I missed that one... the default here is "true" while the default as defined by `REST_EXCEPTION_SKIP_CAUSE_DEFAULT` is `false`... what is the default (I think it should be `false`)? and we should use the constant here as well.

also, when skipping cause, do we want to also skip the whole `root_cause` array?